src/tests/test_rest_framework_dso/test_filters.py

```diff
@@ -42,6 +42,7 @@ class Meta:
                 "name": ["exact"],
                 "category_id": ["exact", "in", "not"],
                 "date_added": ["exact", "lt", "lte", "gt", "gte", "not"],
+                "url": ["exact", "isnull", "not", "isempty", "like"],
```

Comment:
`date_added` and `category_id` have extra tests, maybe good to add same for `url`?

Reply:
Done, in existing test_filter_logic.